popup/popup.js

```diff
@@ -112,8 +112,8 @@ function initPopup(url) {
   }
 
   const u = tryCatch(() => new URL(url));
-  $('#find-styles-link').onclick = handleEvent.openURLandHide;
-  $('#find-styles-link').href +=
+  $('#open-search-link').onclick = handleEvent.openURLandHide;
```

Comment:
the original element id shouldn't be changed; the behavior would be toggled by a checkbox to the right of this link; the second link displayed in the container may have a different id e.g. `find-more-styles-link`